<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patienteditview</class>
 <widget class="QWidget" name="patienteditview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑患者信息</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>80</y>
     <width>314</width>
     <height>225</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>77</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="2" colspan="4">
         <widget class="QComboBox" name="dbComboSex">
          <item>
           <property name="text">
            <string>男</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>女</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="9" column="2" colspan="4">
         <widget class="QLineEdit" name="dbCreatedTimestamp"/>
        </item>
        <item row="2" column="2" colspan="4">
         <widget class="QLineEdit" name="dbEditIDCard"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>身份证：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="4">
         <widget class="QDateEdit" name="dbDateEditDOB"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>姓名:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>体重：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>手机号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>出生日期：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2" colspan="4">
         <widget class="QLineEdit" name="dbEditMobile"/>
        </item>
        <item row="1" column="2" colspan="4">
         <widget class="QLineEdit" name="dbEditName"/>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>性别：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="d">
          <property name="text">
           <string>身高：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="4">
         <widget class="QLineEdit" name="dbEditID">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>创建时间：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" colspan="4">
         <widget class="QSpinBox" name="dbSpinWeight"/>
        </item>
        <item row="5" column="2" colspan="4">
         <widget class="QSpinBox" name="dbSpinHeight"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>150</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
